### PR TITLE
Minor changes and site previews

### PR DESCRIPTION
**Pull request by ThatZiv**
## Describe your changes

Add site screenshots and changed users caching strategy. This also addresses https://github.com/WSU-4110/TrophyTracker/issues/130 bug for incorrect title

## Checklist before requesting a review

- [X] I have performed a self-review of my code.
- [X] There are no lint nor build errors in my code.
- [X] If it is a core feature, I have added thorough tests.

### CODE REVIEW

File: src/app/layout.tsx [hunk 1]
```diff
@@ -17,10 +17,10 @@ const inter = Inter({
   variable: "--font-sans",
 });
 
-// TODO: someone can change this
 export const metadata = {
-  title: "Create T3 App",
-  description: "Generated by create-t3-app",
+  title: "Trophy Tracker",
+  description:
+    "Trophy Tracker is a web app for tracking achievements and trophies in your favorite games.",
```
Comment (XLRA) [inline src/app/layout.tsx:23]: Very nice compact description

**Review by XLRA — APPROVED** [review]
Site previews look good!

Approved.

File: README.md [hunk 2]
```diff
@@ -15,6 +15,128 @@ Team Members:
 
 Meeting [Logs](https://github.com/WSU-4110/Novelology/wiki).
 
+# Screenshots
+
+<details>
+<summary>Home</summary>
+<br>
+
+![home](/.github/preview/ss-home.png)
```
Comment (PierreT20) [inline README.md:24]: LGTM

File: src/app/layout.tsx [hunk 3]
```diff
@@ -17,10 +17,10 @@ const inter = Inter({
   variable: "--font-sans",
 });
 
-// TODO: someone can change this
 export const metadata = {
-  title: "Create T3 App",
```
Comment (PierreT20) [inline src/app/layout.tsx:22]: Finally changed the title

**Review by PierreT20 — APPROVED** [review]

**Pull request merged**